docs/data/sdks/typescript-browser/index.md

```diff
@@ -61,6 +61,10 @@ amplitude.init("YOUR_API_KEY_HERE");
 
 --8<-- "includes/sdk-ts-browser/init.md"
 
+### Logging
```

Comment:
I remember @haoliu-amp suggested to give this a more descriptive name?

Reply:
Do you mean "Debug Mode"? I add a section with that name under this section, https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/pull/401/files#diff-d5a30aad1c3035b9891382e5824ff0909c75032f91760c2bb498a888deb32ec2R27

Reply:
Maybe debuggability?

Reply:
Thanks @yuhao900914 ! I am good with either `Logging` or `Debuggability`, @caseyamp , do you have any suggestion here?

Reply:
I prefer `Logging`  or `Debugging`

Reply:
Thanks @caseyamp, @yuhao900914 , I rename the `Logging` to `Debugging` to you suggested.